app/services/user_profiles_encryptor.rb

```diff
@@ -0,0 +1,28 @@
+class UserProfilesEncryptor
+  attr_reader :personal_key
+
+  def initialize(user, user_session, password)
+    @user = user
+    @user_session = user_session
+    @password = password
+  end
+
+  def call
```

Comment:
Also while we're here, can we find a clearer name than `.call`, how about `.encrypt`?